### DISCUSSION TITLE
Algolia plugin for Pelican

### DISCUSSION THREAD

**Discussion by ataridude** [CloudBytes-Academy/CloudBytes.dev#81] [Q&A] [answered]
Hi. Thanks for publishing your Algolia plugin for Pelican.  I'm in the process of moving my blog to Docker, and while I have Tipue search working today, I haven't been able to get it working in Docker.  Your plugin will be awesome.

I'm following the post I found [here](https://devops.datenkollektiv.de/finally-pelican-with-algolia-search.html), and I'm able to load my site's content into the index. and I'm able to get a search field on the page, but typing into the search field never produces any results.  Any tips on how I could get that going, or any thoughts on what I might be doing wrong here?

Thanks.

<img width="374" alt="Screen Shot 2021-11-04 at 21 50 03" src="https://user-images.githubusercontent.com/87831078/140444842-73ff122b-5bfb-48af-a50e-8b6351501c94.png">

**Comment by rehanhaider** [comment, marked as answer]
Hey, can you paste the HTML code of your search box? I also have updated [this guide](https://cloudbytes.dev/snippets/launching-pelican-algolia-plugin-for-pelican) with instructions to create a search box similar to on [CloudBytes/dev>](https://cloudbytes.dev). 

If this doesn't work, 
1. Paste the HTML code for your search box
2. Paste the JS code that you're using

**Reply by ataridude** [reply]
Yes, I got it now -- I had originally put the JavaScript code at the bottom of the HEAD section in the HTML.  I don't know if your updated guide has new JS code (it looks the same as the original code), or if simply moving the JS code to the bottom of the HTML (just after the `</body>` and before the `</html>`) was what did the trick, but now it works.  Looks like I have some CSS work to do, but I'll be able to get it now - thanks!

**Reply by rehanhaider** [reply]
Congratulations! Could you star the repository if it helped.
